tasks/copy_pyqt_to_venv.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -eu
+
+if [[ ! -z "${VIRTUAL_ENV:-}" ]]; then
```

Comment:
Can use just `[[ -n ... ]]` instead of `[[ ! -z ... ]]`.

Reply:
yep